Winch/Serialization/DredgeTypeConverter.cs

```diff
@@ -80,7 +80,17 @@ protected void AddDefinitions(Dictionary<string, FieldDefinition> definitions)
     {
         foreach (var fieldDefinitionEntry in definitions)
         {
-            this.FieldDefinitions.Add(fieldDefinitionEntry.Key, fieldDefinitionEntry.Value);
+            if (FieldDefinitions.ContainsKey(fieldDefinitionEntry.Key))
+            {
+                this.FieldDefinitions[fieldDefinitionEntry.Key] = new(
+                    fieldDefinitionEntry.Value.DefaultValue,
+                    FieldDefinitions[fieldDefinitionEntry.Key].Parser
+                    );
+            }
```

Comment:
Wanted to be able to overwrite default item values within inherited attributes

Reply:
Just curious: which entries is this applicable for?

Reply:
Not exactly ‘necessary’, but should simplify creating more specific items for the end user- generally just overwriting itemType and Subtype as far as I remember.
E.g putting an item json inside the Relic item folder will give it the appropriate generic types and item color of relics

Reply:
Ah, I see. Thank you for clarifying!